### ISSUE TITLE
Warning with Project-OSRM v0.4.1

### ISSUE THREAD

**Issue by sercoin** [woodbri/osrm-tools#7]
Hi,

I got this warning when I build with latest Project-OSRM (v0.4.1).

```
root@OSRMTemp:/pgr2osrm# make
g++ -g -O0 -I../Project-OSRM/Util  -o pgr2osrm pgr2osrm.cpp ../Project-OSRM/build/libUUID.a -lpqxx -lpq
In file included from pgr2osrm.cpp:5:0:
../Project-OSRM/Util/UUID.h:38:25: warning: defaulted and deleted functions only available with -std=c++11 or -std=gnu++11 [enabled by default]
root@OSRMTemp:/pgr2osrm# 
```

Is it ok?

**Comment by woodbri** [comment]
Sorry, pgr2osrm has not been upgraded to support the v0.4.1 file formats. I just pushed some minor changes which document the current version of Project-OSRM we support.

In general, I am finding that it is very hard to support the OSRM file format because it changes with every OSRM release. Dennis has said the it would be a good idea to support a documented data input API that can be stable in OSRM, but we do not have that yet.

**Comment by sercoin** [comment]
Project-OSRM gives `[warn] Exception occured: negative edge type` error. Is it relevant with this warning?

**Comment by woodbri** [comment]
pgr2osrm does not support Project-OSRM v0.4.1

**Comment by Carsten64** [comment]
Did pgr2osrm support now the version 0.4.1 or higher?

**Comment by woodbri** [comment]
No, and I do not plan to update it until OSRM has a more stable interface for this purpose. You should bug Dennis on this issue https://github.com/DennisOSRM/Project-OSRM/issues/825

**Comment by sercoin** [comment]
Hi,

Is there any update on this?

Regards
